stream-tests/src/test/scala/org/apache/pekko/stream/io/compression/CoderSpec.scala

```diff
@@ -145,6 +147,7 @@ abstract class CoderSpec(codecName: String) extends AnyWordSpec with CodecSpecSu
     }
 
     "be able to decode chunk-by-chunk (depending on input chunks)" in {
+      assume(autoFlush)
```

Comment:
This test is only valid when you flush on each element on the stream as its testing that each individual element can be decompressed on its own and that is not the case when you have `autoFlush` disabled